### ISSUE TITLE
page number buttons are displayed twice 

### ISSUE THREAD

**Issue by keshavsethi** [publiclab/plots2#7530]
In https://publiclab.org/tag/question:general
here page numbers are displaying twice as shown:
![Screenshot from 2020-02-18 16-42-55](https://user-images.githubusercontent.com/36025262/74731492-175be500-526e-11ea-8692-73f84813ba96.png)

**Comment by neelesh17** [comment]
Can I claim this issue?

**Comment by keshavsethi** [comment]
Here Asked and Answered buttons are not working as well thatswhy answered questions are not rendered on clicking thus that div  remain empty and it shows two pagination as shown below 
![Screenshot from 2020-02-20 01-52-47](https://user-images.githubusercontent.com/36025262/74872728-d3f39a80-5383-11ea-96d6-bed98218806b.png)
@cesswairimu @SidharthBansal @VladimirMikulic @nstjean can you please review this issue
Thanks!!

**Comment by VladimirMikulic** [comment]
This is in fact an issue. @neelesh17 you can take this if that's fine by @keshavsethi.

**Comment by cesswairimu** [comment]
Great catch. Thanks

**Comment by keshavsethi** [comment]
@VladimirMikulic actually I was working on this but if @neelesh17 has started this then it is fine :)

**Comment by tyler-wel** [comment]
So I take it progress hasn't been made? If no one is currently looking at it, I can try and take a look 😄

**Comment by avats-dev** [comment]
> So I take it progress hasn't been made? If no one is currently looking at it, I can try and take a look smile

That would be awesome, thanks for stepping up on this. :rocket:

**Comment by tyler-wel** [comment]
Note on the issue

https://publiclab.org/questions/
Questions are displayed correctly with one pagination bar

https://publiclab.org/questions/tag/camera
Displays two pagination bars

Seems to be a problem with the tags routing

**Comment by tyler-wel** [comment]
Both `_questions.html` and `_answered.html` are rendering their `@pagy` pagination.

`_tab_content` has the below `users/answered` partial rendering even though the tab is not active. Also, it seems as if `@answered_questions` exists at this point, but there is no mention of `@answered_questions` inside of `_answered.html`
```
<div class="tab-content">
  <div class="tab-pane active" id="asked-tab">
    <%= render partial: "questions/new_question" %>
  </div>

  <div class="tab-pane" id="answered-tab">
    <% if @answered_questions %>
      <%= render partial: "users/answered" %>
    <% end %>
  </div>
</div>
```

@jywarren I see you did the initial install of pagy and I was wondering what you're thoughts on this are. It seems we may need an extra check in the `<% if @pagy %>` section. I think both the `/questions` and `/questions/tag/{tag}` route are defining their own @pagy and it's creating a little conflict with the two partials

**Comment by tyler-wel** [comment]
@jywarren Upon testing a little more and clicking on the "Asked" and "Answered" tabs on the `questions/tag/` screen, it seems like those two tabs are not working correctly. Nothing changes upon clicking on them. In this case "Test" has been commented on, and "Test2" has not been commented on

![image](https://user-images.githubusercontent.com/49186122/95407628-851d6400-0958-11eb-9b21-4d3a588fe7ea.png)

![image](https://user-images.githubusercontent.com/49186122/95407635-89e21800-0958-11eb-9f4a-ba8cd5b5b0e6.png)

**Comment by jywarren** [comment]
Fixed in #8689 ! Thank you!!!!
